### PR TITLE
Update IP address definition

### PR DESCRIPTION
**Pull request by XmM14**
…th 'files/en-us/glossary/ip_address/'

### CODE REVIEW

File: files/en-us/glossary/ip_address/index.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ tags:
   - Web
 ---
 
-An IP address is a number assigned to every device connected to a network that uses the Internet protocol.
+An IP address accronym for nternet Protocol address, is a unique number assigned to every device connected to a network that uses the Internet protocol. Through this IP address, devices online are able to find each other. Without IP address any of online content couldn't find it's way to us.
```
Comment (dawei-wang) [inline files/en-us/glossary/ip_address/index.md:11]:
```suggestion
An IP address accronym for Internet Protocol address, is a unique number assigned to every device connected to a network that uses the Internet protocol. IP addresses enable devices online to find each other. Without IP addresses we wouldn't be able to view any online content.
```

Comment (XmM14) [reply files/en-us/glossary/ip_address/index.md:11]: It looks absolutely fine! I appreciate your suggestion.

File: files/en-us/glossary/ip_address/index.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ tags:
   - Web
 ---
 
-An IP address is a number assigned to every device connected to a network that uses the Internet protocol.
+An IP address accronym for Internet Protocol address, is a unique number assigned to every device connected to a network that uses the Internet protocol. IP addresses enable devices online to find each other. Without IP addresses we wouldn't be able to view any online content.
```
Comment (hamishwillee) [inline files/en-us/glossary/ip_address/index.md:11]: What value/information does this add over the original text?


- I would remove the last sentence.
- It is perhaps worth saying that the value is unique and used for addressing devices.
- This English is incorrect. 

Minimal changes for English would be:

```suggestion
An **I**nternet **P**rotocol (IP) address is a number used to uniquely address each device on an IP network.
```

Comment (teoli2003) [reply files/en-us/glossary/ip_address/index.md:11]: I agree with @hamishwillee. Maybe we can add the information in a separate sentence:

```suggestion
An **IP address** is a number used to uniquely address each device on an IP network.

_IP_ stands for _Internet Protocol_ which is the [protocol layer](https://docs.oracle.com/cd/E19683-01/806-4075/ipov-7/index.html) the address is associated to.
```

No strong feelings here: I would be perfectly happy with @hamishwillee version.

Comment (hamishwillee) [reply files/en-us/glossary/ip_address/index.md]: Thanks @teoli2003 , I have a very slight preference for your version. Have committed it and will approve. Thanks @XmM14 , @dawei-wang @teoli2003 for making this happen.

**Review by hamishwillee — APPROVED** [review]

**Pull request merged**